README.md

```diff
@@ -1,68 +1,120 @@
 # HTTP API video editor
+Friendly Web HTTP API video editor with pluggable file storages, video editing backends, and streaming capabilities.
 
-Friendly HTTP API video editor with pluggable file storage, video editing backend, and streaming capabilities.
 
-[![Build Status](https://travis-ci.org/superdesk/video-server.svg?branch=master)](https://travis-ci.org/superdesk/video-server)
-[![Coverage Status](https://coveralls.io/repos/github/superdesk/video-server/badge.svg?branch=master)](https://coveralls.io/github/superdesk/video-server?branch=master)
-
-## Main features
-- upload video
+## Features
+- create a project<sup>[1](#project)</sup>
+- list all projects
+- retrieve project details
+- delete project
+- duplicate project
 - edit video:
     * trim
     * rotate
     * scale
     * crop
-- manage video projects:
-    * retrieve
-    * list
-    * create
-    * duplicate
-    * delete
-- capture and save thumbnails for preview and timeline
-- upload custom thumbnail for preview
-- get thumbnails
+- capture a thumbnails for timeline<sup>[2](#timeline)</sup>
+- capture a thumbnail for a preview at a certain position of the video, with optional crop and rotate params
+- upload a custom image file for a preview thumbnail
+- get thumbnails files
+- get video file
 - stream video
 
-### Installing
+<a name="project">1</a>: `project` it's a record in db with metadata about video, thumbnails, version, processing statuses, links to files and etc.   
+<a name="timeline">2</a>: `timeline` is a display of a list of pictures in chronological order. Useful if you build a UI.
+
 
+## Installation & Run
 These services must be installed, configured and running:
 
  * Python (>= 3.6)
  * FFmpeg
  * MongoDB 
  * RabbitMQ (celery backend)
 
-After required services were installed and running, 
-you will need to clone the repo and install python dependencies:
+After required services were installed and started, 
+you can proceed with a video server installation.
 
-NOTE: use [virtualenv](https://docs.python.org/3/library/venv.html) and [pip](https://pypi.org/project/pip/) to install python modules.
+
+### Installation for development
+NOTE: Use [virtualenv](https://docs.python.org/3/library/venv.html) and [pip](https://pypi.org/project/pip/) to install python modules.
 
 ```
 # clone project
-git clone git@github.com:superdesk/video-server.git
+git clone https://github.com/superdesk/video-server.git
+
+# install video server for development
+# NOTE: your virtualenv must be activated
+pip install -e video-server/[dev]
```

Comment:
`pip install -e .[dev]`

Reply:
I think in this case I need to add `cd video-server` otherwise it will not find setup.py

Reply:
right, I was assuming it's in the folder already